src/App.tsx

```diff
@@ -1,23 +1,42 @@
-import { Routes, Route } from 'react-router-dom';
-import { Main } from './page/main/main';
-import About from './page/about/about';
-import Page404 from './page/404/404';
-import { Layout } from './component/Layout';
-
 import React from 'react';
```

Comment:
The name of the file is not logical right. It would be more correct to use Router